README Polish translation

M4Tdev opened the pull request:
## What kind of change does this PR introduce?

Polish translation for README.md.

## What is the current behavior?

NA

## What is the new behavior?

Polish translations. Help for #1341 

## Additional context

Add any other context or screenshots.

M4Tdev committed:
feat: added polish translation for readme

M4Tdev committed:
fix: removed polish translation link from polish translation file

M4Tdev committed:
feat: add link to Polish translation in other readme files

M4Tdev committed:
fix: some minor translation tweaks

M4Tdev committed:
feat: add translations link to polish README file

M4Tdev committed:
Merge branch 'master' into feat/readme-polish-translation

kiwicopple commented:
this looks great @M4Tdev.

It looks like you've modified a few other files - could you please revert those changes? We are now adding the language list here: https://github.com/supabase/supabase/blob/master/i18n/languages.md
(also in the main readme)

M4Tdev committed:
fix: reverted changes in other README translations files

M4Tdev committed:
fix: reverted other README translations to update master branch

M4Tdev committed:
feat: add Polish translation to languages md

M4Tdev commented:
@kiwicopple Hi, I have reverted other files to the current master branch where translations are organized, and I have added a translation link to the languages.md file.

kiwicopple committed:
Merge branch 'master' into feat/readme-polish-translation
